src/Italy/Mews.Fiscalizations.Italy/ElectronicInvoiceFile.cs

```diff
@@ -26,10 +27,8 @@ public ElectronicInvoiceFile(ElectronicInvoice invoice)
 
         private string Serialize(ElectronicInvoice invoice)
         {
-            var xmlNamespaces = new XmlSerializerNamespaces();
-            xmlNamespaces.Add("p", ElectronicInvoice.Namespace);
-
-            var xml = XmlManipulator.Serialize(invoice, xmlNamespaces).OuterXml;
+            var data = new XmlSerializationData(namespaces: new XmlNamespace("p", ElectronicInvoice.Namespace).ToEnumerable());
```

Comment:
What about changing `ElectronicInvoice.Namespace` to `XmlNamespace` directly?

Reply:
ElectronicInvoice.Namespace is used on many places, so changing it to XmlNamespace means changing the usages in all the DTOS where its used. so i wouldn't make this change.